[2025-02-14T23:05:23.565+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T23:05:23.588+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_airflow_roles.get_owners manual__2025-02-14T23:05:21.937919+00:00 [queued]>
[2025-02-14T23:05:23.605+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_airflow_roles.get_owners manual__2025-02-14T23:05:21.937919+00:00 [queued]>
[2025-02-14T23:05:23.606+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-14T23:05:23.627+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): get_owners> on 2025-02-14 23:05:21.937919+00:00
[2025-02-14T23:05:23.635+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1045) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T23:05:23.637+0000] {standard_task_runner.py:72} INFO - Started process 1056 to run task
[2025-02-14T23:05:23.637+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'manage_***_roles', 'get_owners', 'manual__2025-02-14T23:05:21.937919+00:00', '--job-id', '347', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpf8hxs1ip']
[2025-02-14T23:05:23.638+0000] {standard_task_runner.py:105} INFO - Job 347: Subtask get_owners
[2025-02-14T23:05:23.703+0000] {task_command.py:467} INFO - Running <TaskInstance: manage_airflow_roles.get_owners manual__2025-02-14T23:05:21.937919+00:00 [running]> on host 71404c6b5320
[2025-02-14T23:05:23.901+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='manage_***_roles' AIRFLOW_CTX_TASK_ID='get_owners' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T23:05:21.937919+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T23:05:21.937919+00:00'
[2025-02-14T23:05:23.902+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-14T23:05:23.905+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/***/dags
[2025-02-14T23:05:23.920+0000] {python.py:240} INFO - Done. Returned value was: ['data-team', 'teste']
[2025-02-14T23:05:23.944+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-14T23:05:23.944+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=manage_***_roles, task_id=get_owners, run_id=manual__2025-02-14T23:05:21.937919+00:00, execution_date=20250214T230521, start_date=20250214T230523, end_date=20250214T230523
[2025-02-14T23:05:23.973+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T23:05:23.997+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-14T23:05:23.999+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
